nova/nova/serializers.py

```diff
@@ -3,14 +3,14 @@
 from rest_framework import serializers
 
 from .libs.serializers import NovaSerializer
-from .models import User, Article, TradingEquipment, Comment
+from .models import User, Article, TradingEquipment, Comment, Parity
 
 
 class UserSerializer(NovaSerializer):
     class Meta:
         model = User
         fields = ['email', 'lat', 'long', 'first_name', 'last_name', 'date_of_birth', 'profile_image', 'password', 'pk',
-                  'groups', 'email_activated']
+                  'groups']
```

Comment:
Why is this field (email_activated) removed?

Reply:
I did not realize, I will put it back and then commit